script/upgrade_bundler.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+set -e
+
+version="$(grep -m 1 -A 1 -x -F "BUNDLED WITH" Gemfile.lock | tail -n 1  | tr -d '[:space:]')"
```

Comment:
`-m 1`: find the first occurrence
`-A `: print the next line as well
`-x -F`: find exactly that string without interpreting regex
`tail -n 1`: take the last line
`tr -d`: delete all spaces